vite.config.mts

```diff
@@ -21,6 +21,7 @@ if (!appVersion) {
 const __dirname = dirname(fileURLToPath(import.meta.url))
 const cesiumSource = `${__dirname}/node_modules/cesium/Source`
 const cesiumWorkers = '../Build/Cesium/Workers'
```

Comment:
not sure it would also be a good idea to replace this relative path with something similar to the line above

Reply:
but then we need to modify further down. However I'll remove this cesiumWorkers constant as it is used only in one place and use the path hardcoced like for the other assets, it make the code easier to understand.